apollo-router/tests/snapshots/apollo_reports__condition_if.snap

```diff
@@ -11,7 +11,7 @@ header:
   uname: "[uname]"
   executable_schema_id: "[executable_schema_id]"
 traces_per_query:
-  "# -\nquery($if:Boolean!){topProducts{name...@defer(if:$if){reviews{author{name}}reviews{author{name}}}}}":
+  "# -\nquery($if:Boolean!){topProducts{name...@defer(if:$if,label:\"\"){reviews{author{name}}reviews{author{name}}}}}":
```

Comment:
this is generated by the operation_signature function in the query planner, I have not found a way so far to remove it